decidim-admin/app/packs/stylesheets/decidim/admin/modules/_table-list.scss

```
@@ -144,3 +144,7 @@
     display: inline-block;
   }
 }
+
+tbody tr.highlighted{
```

```suggestion
tbody tr.admin-highlighted{
```
Maybe a more specific classname would be better here